incubator/kube-ops-view/templates/_helpers.tpl

```diff
@@ -0,0 +1,16 @@
+{{/* vim: set filetype=mustache: */}}
+{{/*
+Expand the name of the chart.
+*/}}
+{{- define "name" -}}
+{{- default .Chart.Name .Values.nameOverride | trunc 24 | trimSuffix "-" -}}
```

Comment:
@prydonius will do! I guess `helm init` should also be changed as I got it from there.